authentication.py

```diff
@@ -5,23 +5,49 @@
 
 
 class Authentication:
+	
```

Comment:
Could you change your editor's tabs to be 4 spaces, please?
It is more standard and the way the whole project is formatted. Also causes the diff to wrongly show as with many more changes than your actual ones